<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {border-image: url(:/graphics/graphics/graphics/image.png)0 0 0 0 stretch stretch; 
}
#btnLogin,#btnDebit,#btnCredit,#cardnum,#cardpin{
background-color: rgb(0, 0, 0);
font-family:monofonto;
color: rgb(14, 136, 37);
border-style:solid;
border-width: 2px;
border-color: rgb(14, 136, 37);
border-radius: 10px;
padding: 2px 2px;}
QLabel{color: rgb(14, 136, 37);font-family:monofonto;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Kirjaudu</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>monofonto</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KORTTI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>monofonto</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PIN </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cardnum">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>monofonto</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="cardpin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>monofonto</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>monofonto</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Syötä kortin numero ja paina Enter </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCredit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Credit</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDebit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Debit</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
